cocoviewer.py

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env python3
 """COCO Dataset Viewer.
```

Comment:
```suggestion
"""COCO Dataset Viewer.

```
Let's keep an empty line here.